in_progress/mission-statement.md

```diff
@@ -0,0 +1,53 @@
+We are committed to keeping this community open to third-party contributions and usage and are committed to acknowledging contributions.
+This white paper may serve as an entry point for third parties into the structure of the scverse environment.
+
+## User engagement
+
+The ecosystem centred on `scverse` core packages consists of python packages which are documented on readthedocs and commonly accompanied by scientific publications.
+Its primary purpose is to allow users to generalise workflows to their own data analysis settings.
+We promote user engagement via the following mechanisms:
+
+* Clear documentation of code and usage of tutorial notebooks.
+* Participation in development though issues and pull requests.
+* Community forums, such as our Discourse and Twitter account. scverse environment-centric workshops at which scverse workflows are taught and hackathons at which functionalities are contributed. We share event dates and resources within the environment to increase impact and to exploit synergies.
+
+## Developer engagement
+
+The scverse heavily benefits from a variety of people contributing code via different channels. The channels used at the moment are:
+
+* Pull requests on scverse core or ecosytem packages: These can be used to fix bugs that are experienced during a workflow or to contribute functionalities. 
+* Feedback/ bug reporting through issues and the community forum.
+* Creating and sharing tutorial content, either through written documentation or in person workshops.
+* Contributing and maintaining documentation, making it easier for people to use or understand available tools.
+* Participation in community forums, by helping new users or participating in discussions.
+* Ecosystem packages: Development of tools and analyses which interact with core packages, work with AnnData or MuData objects, or further what can be done with single cell data in Python.
+
+As a community, we try to incentivise code contributions through the following mechanisms:
+
+* Pull requests: New workflows or methods can be contributed to the existing packages as extra functionalities in a pull request. These functionalities can be documented as being advised to be cited separately from the target repository if they are referenced in a scientific publication. Examples that were contributed to Scanpy are documented here: https://scanpy.readthedocs.io/en/docsearch/external/index.html. 
+* Promotion of ecosystem packages: These packages can and are cited separately from Scanpy and can therefore further the developers academic careers.  Secondly, these packages are advertised on [scverse.org](), thus increasing the user base that is reached with the new package.
+* Core engagement in the community is acknowledged as contribution mentions on the website, which may certify the engagement in this community. Moreover, we can attribute contributions in our Twitter accounts to make this readily available for your career development.
```

Comment:
only Twitter for career development?

Reply:
Think that @falexwolf is maintaining the linkedin? Could we rebrand the Scanpy Linkedin to scverse?

Reply:
* I would remove the career development statement, I feel it's irrelevant for the mission statement. 
 * This block feels partly redundant to the block above. Maybe we can first describe the code and ecosystem contributions, and then continue with the block above with something like 
 
 > Apart from direct contributions of code through pull-requests to scverse core or ecosystem packages, we emphasize the importance of

Reply:
this conversation is outdated as the section has been removed, will resolve